swap_meet/item.py

```diff
@@ -1,2 +1,31 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, age=0):
+        self.id = uuid.uuid4().int if id is None else id
+        self.condition = condition
+        self.age = age
+
+
+    def get_category(self):
+        return "Item"
+
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+        
+    def condition_description(self):
```

Comment:
👍 Because all the subclasses are specified to share the same description behavior, we can implement this behavior here in `Item` alone, then leave it to inheritance to make this available to all of the subclasses.